src/drivers/drv_rgb_led.h

```diff
@@ -3,4 +3,8 @@
 void rgb_init();
 void rgb_send(int data);
 
-void rgb_dma_start();
\ No newline at end of file
+void rgb_dma_start();
+void rgb_initIO();
```

Comment:
internal functions in header file - these are not meant to be used by the rest of the codebase, please remove them here.
you might have to reorder you functions in the c file.